### ISSUE TITLE
Crash when using Blame/Timeline

### ISSUE THREAD

**Issue by NeilBurkholder** [ForkIssues/TrackerWin#1510]
When opening the Blame/Time line window for one of my repositories I get a crash. Also for the same repo the history window is blank.
For another repo both these features work as expected.

![image](https://user-images.githubusercontent.com/24275193/163365039-0066f802-3f16-495f-817d-035c8a28809a.png)

**Comment by DanPristupov** [comment]
Can you open the history window and then send me the log file (`%localappdata%\fork\logs\fork.log`) to support@fork.dev?

**Comment by NeilBurkholder** [comment]
After looking at the fork.log I was able to find and fix the problem. It would be nice though to have a relevant error msg rather than a crash though.

 https://stackoverflow.com/a/13918515/718246

**Comment by NeilBurkholder** [comment]
```
?? 2022-04-14 08:15:48.694 Looking for an associated editor for 'C:\Users\NeilHome\source\repos\Building Creator\RBScripts/Building_Creator/lib/specs.js'
?? 2022-04-14 08:15:48.694 File can be edited with 'C:\Windows\System32\WScript.exe'
?? 2022-04-14 08:15:59.559 Git request failed'-c core.quotePath=false log --no-show-signature --oneline --name-status --follow --pretty=format:%H%n%P%n%aN%n%aE%n%at%n%s 676ebd99bf4c2f7c30b54487d7ce8967658ed607 -- "RBScripts/Building_Creator/lib/specs.js"':
error: inflate: data stream error (unknown compression method)
error: unable to unpack 7ed075499fdd7c03a2cbed45e7a88e33e1c57463 header
fatal: loose object 7ed075499fdd7c03a2cbed45e7a88e33e1c57463 (stored in .git/objects/7e/d075499fdd7c03a2cbed45e7a88e33e1c57463) is corrupt
```

**Comment by DanPristupov** [comment]
Thank you, we'll fix this in 1.72.
